include/swift/AST/ASTWalker.h

```diff
@@ -20,14 +20,17 @@
 namespace swift {
 
 class ArgumentList;
+class ASTContext;
```

Comment:
Are these changes to ASTWalker.h still needed, since there were no other changes in this file?